--- lsvpd-1.6.11/Makefile.am.orig	2011-06-16 00:59:59.000000000 +0200
+++ lsvpd-1.6.11/Makefile.am	2011-09-23 19:29:44.906120445 +0200
@@ -40,7 +40,7 @@
 lsmcode_SOURCES = src/output/lsmcode.cpp \
 		$(update_h_files)
 
-vpdupdate_LDADD = -lz -lstdc++ -lvpd_cxx -lsgutils
+vpdupdate_LDADD = -lz -lstdc++ -lvpd_cxx -lsgutils2
 vpdupdate_LDFLAGS = -Wall
 lsvpd_LDADD = -lz -lstdc++ -lvpd_cxx
 lsvpd_LDFLAGS = -Wall
@@ -53,25 +53,9 @@
 
 AM_CXXFLAGS = -DDEST_DIR='"${exec_prefix}"' -DVPD_VERSION='"@VERSION@"'
 
-if PCI_IDS_LOC
-AM_CXXFLAGS += -DPCI_IDS='"/usr/share/pci.ids"'
-else
-if PCI_IDS_LOC2
-AM_CXXFLAGS += -DPCI_IDS='"/usr/share/hwdata/pci.ids"'
-else
-AM_CXXFLAGS += -DPCI_IDS='"/usr/share/misc/pci.ids"'
-endif
-endif
-
-if USB_IDS_LOC
-AM_CXXFLAGS += -DUSB_IDS='"/usr/share/usb.ids"'
-else
-if USB_IDS_LOC2
-AM_CXXFLAGS += -DUSB_IDS='"/usr/share/hwdata/usb.ids"'
-else
-AM_CXXFLAGS += -DUSB_IDS='"/usr/share/misc/usb.ids"'
-endif
-endif
+AM_CXXFLAGS += -DPCI_IDS='"/etc/pci.ids"'
+
+AM_CXXFLAGS += -DUSB_IDS='"/etc/usb.ids"'
 
 SUBDIRS = docs
 
