--- lsvpd-1.6.10/Makefile.am.orig	2010-10-16 01:18:16.000000000 +0200
+++ lsvpd-1.6.10/Makefile.am	2011-05-02 20:46:42.634706026 +0200
@@ -40,7 +40,7 @@
 lsmcode_SOURCES = src/output/lsmcode.cpp \
 		$(update_h_files)
 
-vpdupdate_LDADD = -L. -lz -lstdc++ -lvpd_cxx -lsgutils
+vpdupdate_LDADD = -L. -lz -lstdc++ -lvpd_cxx -lsgutils2
 vpdupdate_LDFLAGS = -Wall
 lsvpd_LDADD = -L. -lz -lstdc++ -lvpd_cxx
 lsvpd_LDFLAGS = -Wall
@@ -53,25 +53,9 @@
 
 AM_CXXFLAGS = -DDEST_DIR='"${exec_prefix}"' -DVPD_VERSION='"@VERSION@"'
 
-if PCI_IDS_LOC
-AM_CXXFLAGS += -DPCI_IDS='"/usr/share/pci.ids"'
-else
-if PCI_IDS_LOC2
-AM_CXXFLAGS += -DPCI_IDS='"/usr/share/hwdata/pci.ids"'
-else
-AM_CXXFLAGS += -DPCI_IDS='"/usr/share/misc/pci.ids"'
-endif
-endif
+AM_CXXFLAGS += -DPCI_IDS='"/etc/pci.ids"'
 
-if USB_IDS_LOC
-AM_CXXFLAGS += -DUSB_IDS='"/usr/share/usb.ids"'
-else
-if USB_IDS_LOC2
-AM_CXXFLAGS += -DUSB_IDS='"/usr/share/hwdata/usb.ids"'
-else
 AM_CXXFLAGS += -DUSB_IDS='"/usr/share/misc/usb.ids"'
-endif
-endif
 
 SUBDIRS = docs
 
